games-board/netmaumau/netmaumau-0.16-r1.ebuild

```diff
@@ -32,6 +32,10 @@ DEPEND="
 
 S=${WORKDIR}/${P}-client
 
+src_prepare() {
+	epatch "${FILESDIR}/connect-to-old-servers.patch"
```

Comment:
Reject that PR, as I told I won't fix it. Doesn't matter but that git f*ck is 
annoying me too much